src/components/App.jsx

```diff
@@ -1,14 +1,183 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
 import { hot } from 'react-hot-loader'
 
 import config from 'config/configFromCookie'
+import 'styles/qtcstyles.scss'
+//const amidaLogo = require('../assets/amida-logo.png');
+//import logo from '../assets/amida-logo.png';
 
 const { applicationPresentationName } = config
+//const API_URL = process.env.REACT_APP_API_URL;
+const API_URL = 'http://ec2-54-144-23-157.compute-1.amazonaws.com/'
 
-// Just need to import this once somewhere so it ends up in the webpack dependency graph.
-// eslint-disable-next-line no-unused-vars
-import styles from 'styles/styles.scss'
+/*
+ * IndexPage: Welcome/home page.
+*/
+function IndexPage() {
+    return (
+        <div className='app-page' id='index-page-div'>
+            <h2>Welcome!</h2>
+            You've reached the QTC Mapping-DBQs API.
+            Please select a page from the navigation menu above in order to create a mapping, 
+            generate a mapping in dev mode, or run tests on a pre-mapped DBQ.
+            Please reach out with any questions or concerns.
+        </div>
+    )
+}
 
-const App = () => <div>This is {applicationPresentationName}</div>
+/*
+ * CreatePage: React frontend for --create functionality.
+ *
+ * Prompts the user for a DBQ number, calls the corresponding mapping_data_frames 
+ * API endpoint, and downloads the resulting file.
+*/
+class CreatePage extends Component {
```

Comment:
I think from a readability perspective we should move this Component `CreatePage` to it's own file. Something like `src/components/CreatePage.jsx`

Reply:
done!